### PR TITLE
Implement scheduler to change learning rate

### PR DESCRIPTION
**Pull request by abelsiqueira**
**Description**

Implementing the famous scheduler. Looks like there are many options to choose from: https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
These should probably be added to the extra information on the log string.

**Related issues**:

- ...

**Instructions to review the pull request**

### CODE REVIEW

**Comment by bart1** [comment]
By default now the learning rate was decreased a factor of 10 at the first iteration that seemed strange, I have also added logging
